x-pack/plugins/security_solution/public/cloud_posture/pages/findings/rule_flyout.tsx

```diff
@@ -22,166 +22,21 @@ import {
-            { title: 'Platform', description: data.host.os.platform },
-            { title: 'Type', description: data.host.os.type },
-            { title: 'Version', description: data.host.os.version },
-          ]}
-        />
-      </EuiCard>
-    </div>
-  );
-};
-const RuleTab = ({ data }: { data: any }) => {
-  return (
-    <div style={{ display: 'grid', gridGap: 15, gridAutoFlow: 'rows' }}>
-      <EuiCard
-        textAlign="left"
-        title={<CardTitle title="Rule" />}
-        style={{ gridRow: 1, gridColumn: 1 }}
-      >
-        <EuiDescriptionList
-          compressed={false}
-          type="column"
-          listItems={[
-            { title: 'Benchmark', description: data.rule.benchmark },
-            { title: 'Name', description: data.rule.name },
-            { title: 'Description', description: data.rule.description },
-            { title: 'Tags', description: getTagsBadges(data.rule.tags) },
-            { title: 'Remediation', description: <EuiCode>{data.rule.remediation}</EuiCode> },
-          ]}
-        />
-      </EuiCard>
-    </div>
-  );
-};
-
-const ResourceTab = ({ data }: { data: any }) => {
-  return (
-    <div style={{ display: 'grid', gridGap: 15, gridAutoFlow: 'rows' }}>
-      <EuiCard
-        textAlign="left"
-        title={<CardTitle title="Resource" />}
-        style={{ gridRow: 1, gridColumn: 1 }}
-      >
-        <EuiDescriptionList
-          compressed={false}
-          type="column"
-          listItems={[
-            { title: 'Filename', description: <EuiCode>{data.resource.filename}</EuiCode> },
-            { title: 'GID', description: data.resource.gid },
-            { title: 'Mode', description: data.resource.mode },
-            { title: 'Path', description: <EuiCode>{data.resource.path}</EuiCode> },
-            { title: 'Type', description: data.resource.type },
-            { title: 'UID', description: data.resource.uid },
-          ]}
-        />
-      </EuiCard>
-      <EuiCard
-        textAlign="left"
-        title={<CardTitle title="Result" />}
-        style={{ gridRow: 2, gridColumn: 1 }}
-      >
-        <EuiDescriptionList
-          compressed={false}
-          type="column"
-          listItems={[
-            { title: 'Evaluation', description: getEvaluationBadge(data.result.evaluation) },
-            {
-              title: 'Evidence',
-              description: <EuiCode>{JSON.stringify(data.result.evidence, null, 2)}</EuiCode>,
-            },
-          ]}
-        />
-      </EuiCard>
-    </div>
-  );
-};
-
 const tabs = ['resource', 'rule', 'overview'] as const;
 
-export const FindingsRuleFlyOut = ({
-  onClose,
-  findings,
-}: {
+interface CSPTabProps {
+  data: CSPFinding;
+}
+
+interface FindingFlyoutProps {
   onClose(): void;
   findings: CSPFinding;
-}) => {
-  console.log({ findings });
-  const [tab, setTab] = useState('resource');
+}
+
+export const FindingsRuleFlyOut = ({ onClose, findings }: FindingFlyoutProps) => {
```

Comment:
nitpick, but it seems the flyout component is being mentioned as `Flyout` in most EUI and your types here
so maybe you want 
```suggestion
export const FindingsRuleFlyout = ({ onClose, findings }: FindingFlyoutProps) => {
```